development/docker-compose.receptor_mesh.yml

```diff
@@ -0,0 +1,57 @@
+services:
+  remote-worker:
```

Comment:
remote-worker service generated from nautobot image. `node01` gets spun up inside the container as well to interact with `node02` and `node03`